PowerEditor/src/DarkMode.cpp

```diff
@@ -0,0 +1,291 @@
+#pragma once
```

Comment:
This is not needed in .cpp

Reply:
> Also, you should add a GPL comment header as in other source files.

And @adzm 's name (or pseudo) and email - It would be great but it's not mandatory.